flink-python/pyflink/datastream/tests/test_stream_execution_environment.py

```diff
@@ -216,7 +216,7 @@ def test_execute(self):
 
     def test_from_collection_without_data_types(self):
         ds = self.env.from_collection([(1, 'Hi', 'Hello'), (2, 'Hello', 'Hi')])
-        collect_util = DataStreamCollectUtil()
+        collect_util = DataStreamTestSinkFunction()
```

Comment:
The test would fail, i.e., there is no `collect` method on `collect_util`.

Reply:
Sorry, I forget to update this part.